substrate-metadata/src/util.ts

```diff
@@ -210,3 +211,41 @@ export function isPreV14(metadata: Metadata): boolean {
             return false
     }
 }
+
+interface PolkadotJSTypesBundle {
+    spec: {
+        [specName: string]: {
+            types: {
+                types: Record<string, OldTypeDefinition>
+                minmax: [
+                    min: number | undefined | null,
+                    max: number | undefined | null
+                ]
+            }[]
+            alias: OldTypesAlias,
+            signedExtensions: Record<string, OldTypeExp>
+        }
+    }
+}
+
+export function convertPolkadotJSTypesBundle(typesByndle: PolkadotJSTypesBundle): Record<string, OldTypesBundle> {
+    let res: Record<string, OldTypesBundle> = {}
+
+    for (let specName in typesByndle.spec) {
+        let spec = typesByndle.spec[specName]
+        res[specName] = {
+            types: {},
+            typesAlias: spec.alias,
+            signedExtensions: spec.signedExtensions,
```

Comment:
Our signed extensions are different.

Reply:
@eldargab should I convert polkadot.js signedExtensions to ours?